src/Form/Input/Input.component.tsx

```diff
@@ -62,10 +57,28 @@ export const StyledInput = styled.div `
     }
   }
 
+  .field-input {
+    flex: 1;
+    overflow: hidden;
+    width: 100%;
+    padding: 24px 0 6px ${({ prefix }: InputProps) => !prefix ? '.75rem' : 'calc(1rem - 3px)'};
```

Comment:
This doesn't seem quite right.

Syntax:
`[rule]: ${(props) => props ? 'value if props' : 'value if !props'};`

Here:
`padding: 24px 0 6px ${({ prefix }: InputProps) => !prefix ? '.75rem' : 'calc(1rem - 3px)'};`

We define a padding for everything but the `padding-left`. I think I'd prefer:

`padding: ${({ prefix }: InputProps) => !prefix ? '1.5rem 0 0.375rem .75rem' : '1.5rem 0 0.375rem calc(1rem - 3px)'};`

This solves mixed units and it's a little bit easier to read. It's not wrong, though. This just seems tidier. What do you think?

Reply:
Should we define the width property using this calculation as well? Otherwise, it could be 100% + the padding values and overflow parent elements.

Reply:
This width is only for the input part of this component. Currently 100% means it's 100% to the `container - prefix`. Why do you think it can be overflow here?

Reply:
This component has a width of 100% and a padding applied which makes it larger than a containing element I think. [Reference 1](https://stackoverflow.com/questions/4879788/padding-adds-to-div-width-height)

Reply:
what's the idea re: _not_ opting into `border-box` sizing?

Reply:
Asking the real questions. I had originally added it to the normalize global override, but I wasn't sure what our internal teams would be familiar with. I'd gladly re-introduce this if it makes calculation easier for our eng team, but not if it creates another obstacle. @knksmith57 and @Michelle-shw what's your preference here? Consider this an informal poll. =P

Reply:
unless existing code relies specifically on `content-box` behavior (which would surprise me), my gut says we should absolutely prefer `border-box`.